### ISSUE TITLE
Can you give me some suggestions about fine tune squad models in your project ?

### ISSUE THREAD

**Issue by svjack** [abhijithneilabraham/tableQA#54]
As before discussion, i have try to replace the function in other language.
And i think about fine tuning the squad model you use to extract condition string
from question input (as the code says, you use colquery (construct by keyword and use “which” 
or “number of” as question word) as question and truly input question as document to extract.
If one want to have a better inference on this, should have a fine tuning on its dataset.
So can you give me some suggestions about labeling myself datasets for fine tuning ?
I think if i always use your colquery construction to construct my squad dataset may be too plain.

**Comment by svjack** [comment]
And i think use named entity recognition and relation extraction can extract the conditions more accurately.

**Comment by abhijithneilabraham** [comment]
Yes what you said is correct.  We are using a bert model from huggingface for the QA task. Since you are doing chinese QA, you actually have to use a chinese QA model, see if it's already available from huggingface, or you can find a dataset and train a QA model on top of it.
